src/ComplexScalars/UriTrait.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Created by PhpStorm.
+ * User: kevbaldwyn
+ * Date: 14/02/2018
+ * Time: 14:58
+ */
+
+namespace Funeralzone\ValueObjectExtensions\ComplexScalars;
+
```

Comment:
The PSR 2 check is complaining because there are two lines here.

If you want to check locally you can do `./vendor/bin/phpcs -n --standard=PSR2 --ignore=vendor/ .`